set-web/src/main/java/eu/europeana/set/web/service/controller/jsonld/WebUserSetRest.java

```diff
@@ -457,7 +466,8 @@ private void validateAndSetItems(UserSet storedUserSet, UserSet updateUserSet, L
 	    checkIfMatchHeader(eTagOrigin, request);
 
 	    UserSet updatedUserSet = getUserSetService().insertItem(datasetId, localId, position, existingUserSet);
-	    String serializedUserSetJsonLdStr = serializeUserSet(profile, updatedUserSet);
+	    UserSet resUserSet = setTotalField(profile, updatedUserSet);
```

Comment:
move to applyProfile method as indicated in previous comment

Reply:
functionality moved to applyProfile method